src/solid_poro_3D_ele/4C_solid_poro_3D_ele_pressure_velocity_based.cpp

```diff
@@ -42,6 +45,18 @@ namespace Discret::Elements::SolidPoroPressureVelocityBasedInternal
           entry<std::vector<double>>("POROANISODIR1", {.required = false, .size = 3}),
           entry<std::vector<double>>("POROANISODIR2", {.required = false, .size = 3}),
           entry<std::vector<double>>("POROANISODIR3", {.required = false, .size = 3}),
+          Inpar::ScaTra::get_impl_type_input_spec("TYPE",
```

Comment:
As I said, I don't see the advantage of the function over simply creating the map here.